readme.md

```diff
@@ -17,7 +17,9 @@ Using jQuery, write code that makes an AJAX get request to "http://kittengifs.co
 
 Your Answer:
 ```js
-
+$.ajax({url: "http://kittengifs.com/gifs", success: function(response){
+  console.log($(response).length)
+});
```

Comment:
Think this would work without the $() around response. We've talked about handling ajax callback a little differently: https://github.com/ga-dc/curriculum/tree/master/07-apis-express-ajax/ajax#ajax-2550